.buildkite/react-native-cli-pipeline.yml

```diff
@@ -708,26 +708,26 @@ steps:
         concurrency_group: "browserstack-app"
         concurrency_method: eager
 
-      - label: ":runner: RN 0.67 Android end-to-end tests (Hermes)"
-        depends_on: "rn-cli-0-67-hermes-apk"
-        timeout_in_minutes: 10
-        plugins:
-          artifacts#v1.5.0:
-            download: "build/rn0_67_hermes.apk"
-            upload: ./test/react-native-cli/maze_output/**/*
-          docker-compose#v4.12.0:
-            pull: react-native-cli-maze-runner
-            run: react-native-cli-maze-runner
-            use-aliases: true
-            command:
-              - --app=build/rn0_67_hermes.apk
-              - --farm=bs
-              - --device=ANDROID_12_0
-              - --a11y-locator
-              - features/run-app-tests
-        concurrency: 5
-        concurrency_group: "browserstack-app"
-        concurrency_method: eager
+#      - label: ":runner: RN 0.67 Android end-to-end tests (Hermes)"
```

Comment:
Remove Hermes tests for 0.67 as this is officially not supported now.

(Also agreed to introduce a Hermes enabled 0.69 test.)